src/Implementation/Output.php

```diff
@@ -6,35 +6,71 @@
 
 class Output implements OutputInterface
 {
+    /**
```

Comment:
the formats might be better split into colors and others, so that error is (for example) selff::$red.self::$bold